fix(nextjs): Post-release cleanup (#3435)
getsentry/sentry-docs: src/includes/getting-started-config/javascript.nextjs.mdx

> Though it's possible to [configure everything manually](/platforms/javascript/guides/nextjs/manual-setup/), we have a wizard which will automate the initial steps. To use it, run the following from the root level of your project:

Reviewer: ```suggestion
Though it's possible to [configure everything manually](/platforms/javascript/guides/nextjs/manual-setup/), we have a wizard that will automate the initial steps. To use the wizard, run the following command from the root level of your project:
```

```bash
npx @sentry/wizard -i nextjs
```

> The wizard will prompt you to log into Sentry, and then will automatically add the necessary configuration files to your project. It will perform the following changes:

Reviewer: ```suggestion
You'll be prompted to log in to Sentry. The wizard will then automatically add these configuration files to your project:
```

- create `sentry.client.config.js` and `sentry.server.config.js` with the default `Sentry.init`
- create `next.config.js` with the default configuration
- create `sentry.properties` with configuration for `sentry-cli` (which is used when automatically uploading source maps)

> See the docs on [manual configuration](/platforms/javascript/guides/nextjs/manual-setup/) for examples of the files created by the wizard. If any of these files already exists, it will be created with a leading underscore and you will need to merge it with the existing file manually.

Reviewer: ```suggestion
See [manual configuration](/platforms/javascript/guides/nextjs/manual-setup/) for examples of the files created by the wizard. If any of these files already exists, it will be created with a leading underscore and you will need to merge it with the existing file manually.
```
